<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindFriend</class>
 <widget class="QWidget" name="FindFriend">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>133</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolButton{
	border-style:none;
	font: 12pt &quot;幼圆&quot;;
}
QToolButton:hover{background-color:rgb(201, 218, 255)
}

QLabel{
	
	font: 12pt &quot;幼圆&quot;;
}


QLineEdit{
	
	font: 12pt &quot;幼圆&quot;;
}</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>56</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:rgb(0, 0, 0);
border-color:#FFFFFF;
border-style:solid;
border-top-width:0px;
border-right-width:0px;
border-bottom-width:1px;
border-left-width:0px;</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="1.qrc">
     <normaloff>:/Image/Find.png</normaloff>:/Image/Find.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>好友ID查找</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>0</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="1.qrc">
     <normaloff>:/Image/jianshao.png</normaloff>:/Image/jianshao.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>0</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="icon">
    <iconset resource="1.qrc">
     <normaloff>:/Image/delete.png</normaloff>:/Image/delete.png</iconset>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-20</y>
     <width>401</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border-radius:15px;
border-image: url(:/Image/watercolor-4117017_1280_3.png);</string>
   </property>
  </widget>
  <zorder>widget</zorder>
  <zorder>lineEdit</zorder>
  <zorder>toolButton</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>toolButton_3</zorder>
  <zorder>toolButton_4</zorder>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>toolButton</tabstop>
 </tabstops>
 <resources>
  <include location="1.qrc"/>
 </resources>
 <connections/>
</ui>
